.docker/odoo/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:18.04
 RUN apt update; \
     DEBIAN_FRONTEND=noninteractive apt -y --no-install-recommends install \
       ca-certificates curl gnupg lsb-release python3-pip python3-dev python3-lxml libsasl2-dev libldap2-dev ssh \
-      libjpeg-dev libpq-dev libssl1.0-dev build-essential python3-matplotlib nodejs-dev node-gyp npm nginx git locales;  \
```

Comment:
when you choose `nginx` boolean option from runbot.repo the runbot starts a nginx process

So, I'm not sure if we should remove it

Reply:
True although I dont think it is setup so if enabled it would not work :thinking:, for instance port 80 is already used by the nginx container itself